README.md

```diff
@@ -107,3 +120,60 @@ $ irb
 information that can make cracking the value orders of magnitude easier for an
 attacker. For this reason we advise you not to store the results of
 `Zxcvbn::Tester#test`. Further reading: [A Tale of Security Gone Wrong](http://gavinmiller.io/2016/a-tale-of-security-gone-wrong/).
+
+## Contact
+
+ - [GitHub project](https://github.com/envato/zxcvbn-ruby)
+ - Bug reports and feature requests are welcome via [GitHub Issues](https://github.com/envato/zxcvbn-ruby/issues)
+
+## Maintainers
+
+ - [Pete Johns](https://github.com/johnsyweb)
+ - [Steve Hodgkiss](https://github.com/stevehodgkiss)
+
+## Authors
+
+ - [Steve Hodgkiss](https://github.com/stevehodgkiss)
+ - [Matthieu Aussaguel](https://github.com/matthieua)
+ - [_et al._](https://github.com/envato/zxcvbn-ruby/graphs/contributors)
+
+## License [![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://github.com/envato/zxcvbn-ruby/blob/master/LICENSE.txt)
+
+`zxcvbn-ruby` uses MIT license, the same as [zxcvbn.js][zxcvbn.js] itself. See
+[`LICENSE.txt`](https://github.com/envato/zxcvbn-ruby/blob/master/LICENSE.txt)
+for details.
+
+## Code of Conduct
+
+We welcome contribution from everyone. Read more about it in
+[`CODE_OF_CONDUCT.md`](https://github.com/envato/zxcvbn-ruby/blob/master/CODE_OF_CONDUCT.md).
+
+## Contributing [![PRs welcome](https://img.shields.io/badge/PRs-welcome-orange.svg?style=flat-square)](https://github.com/envato/zxcvbn-ruby/issues)
+
+For bug fixes, documentation changes, and features:
+
+1. [Fork it](./fork)
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Push to the branch (`git push origin my-new-feature`)
+1. Create a new Pull Request
+
+For larger new features: Do everything as above, but first also make contact with the project maintainers to be sure your change fits with the project direction and you won't be wasting effort going in the wrong direction.
```

Comment:
Should we explain how to contact the project maintainers?

Reply:
Is this not covered by 670e217 and a7f6ab5 ?

Reply:
Yeah that feels sufficient to me.

Reply:
I meant to add a link here :)

```suggestion
For larger new features: Do everything as above, but first also [make contact with the project maintainers](https://github.com/envato/zxcvbn-ruby#contact) to be sure your change fits with the project direction and you won't be wasting effort going in the wrong direction.
```